src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
+@import 'https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap';
+@import 'card';
+@import 'stars';
```

Comment:
Ensure that SCSS variables are used for main values and placed in a separate file as specified in the task requirements. This will help in maintaining consistency across your styles.